scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -232,8 +232,8 @@
                                                                                 :gpu (util/same-user-task-comparator))) task-ents)]))
                                             (into {}))
         task->scored-task (into (pm/priority-map-keyfn (case category
-                                                         :normal (comp - :dru)
-                                                         :gpu -))
+                                                         :normal (comp (fnil - 0) :dru)
```

Comment:
Good question.  I only added this for consistency with the change to handle nil gpu's below it.  DRU should never be nil, I'll remove that check.